lib/src/main/kotlin/com/swmansion/starknet/data/types/transactions/Transaction.kt

```diff
@@ -146,21 +170,6 @@ data class InvokeTransactionV1(
             version = version,
         )
     }
-
-    companion object {
```

Comment:
is this no longer needed?

Reply:
We don't use it anywhere, I'm not sure if we ever did use it. I couldn't find it being used in #197 (PR where it was added). Since this is an internal method and we don't use it anywhere internally, I think we don't need it.